src/runtime/pyobject.cs

```diff
@@ -915,6 +915,34 @@ public override bool TrySetMember(SetMemberBinder binder, object value)
             return true;
         }
 
+        private void GetArgs(object[] inargs, CallInfo callInfo, out PyTuple args, out PyDict kwargs)
+        {
+            if (callInfo == null || callInfo.ArgumentNames.Count == 0)
+            {
+                GetArgs(inargs, out args, out kwargs);
+                return;
+            }
+
+            // Support for .net named arguments
+            var namedArgumentCount = callInfo.ArgumentNames.Count;
+            var regularArgumentCount = callInfo.ArgumentCount - namedArgumentCount;
+
+            var argTuple = Runtime.PyTuple_New(regularArgumentCount);
+            for (int i = 0; i < regularArgumentCount; ++i)
+            {
+                AddArgument(argTuple, i, inargs[i]);
+            }
+            args = new PyTuple(argTuple);
+
+            var namedArgs = new object[namedArgumentCount * 2];
```

Comment:
yes, but this way I'm reusing the validation and conversion in the kw method: 
```
...
if (kv[i + 1] is PyObject)
{
    value = ((PyObject)kv[i + 1]).Handle;
}
else
{
    value = Converter.ToPython(kv[i + 1], kv[i + 1]?.GetType());
}
...
```
Maybe the conversion/validation code could be refactored and called by the 2 methods, I have to look into it.